SayTheirNamesTests/ServiceTests/NetworkRequestorTests.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  NetworkRequestorTests.swift
+//  SayTheirNamesTests
+//
+//  Created by Oliver Binns on 05/06/2020.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import Alamofire
+import Mocker
+import XCTest
+@testable import Say_Their_Names
+
+final class NetworkRequestorTests: XCTestCase {
+
+    private struct TestObject: Decodable {
+        let data: String
+    }
+
+    var sut: NetworkRequestor!
+
+    override func setUp() {
+        super.setUp()
+
+        let configuration = URLSessionConfiguration.af.default
+        configuration.protocolClasses = [MockingURLProtocol.self]
+        let session = Session(configuration: configuration)
+        sut = NetworkRequestor(session: session)
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        sut = nil
+    }
+
+    func test_fetchDecodable_returnsObject() {
+        guard let url = URL(string: "https://example.com") else { XCTFail("URL was not valid"); return }
```

Comment:
Maybe we should check `Environment.serverURLString` here?

Reply:
Done ✅